src/Symfony/Component/Process/Exception/ProcessSignaledException.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Process\Exception;
+
+use Symfony\Component\Process\Process;
+
+/**
+ * Exception that is thrown when a process has been signaled.
+ *
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class ProcessSignaledException extends RuntimeException
+{
+    private $process;
+
+    public function __construct(Process $process)
+    {
+        $this->process = $process;
+
+        parent::__construct(sprintf('The process has been signaled with signal "%s".', $process->getTermSignal()));
+    }
+
+    public function getProcess()
```

Comment:
missing return type, isn't it?

Reply:
Well, I did a copy/paste from `ProcessFailedException` for that.

Plus, see this comment: https://github.com/symfony/symfony/pull/25775#pullrequestreview-88457361

I'm not sure I have to add it.

Reply:
Or maybe are you talking about PHP7 return type?

Reply:
yes, I mean `public function getProcess(): Process`, sorry for the confusion